src/web/scraping/extract-links.md

```diff
@@ -9,31 +9,25 @@ Call [`filter_map`] on the [`Selection`] retrieves URLs
 from links that have the "href" [`attr`] (attribute).
 
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
 extern crate reqwest;
 extern crate select;
 
 use select::document::Document;
 use select::predicate::Name;
-#
-# error_chain! {
-#    foreign_links {
-#        ReqError(reqwest::Error);
-#        IoError(std::io::Error);
-#    }
-# }
-
-fn main() -> Result<()> {
-    let res = reqwest::get("https://www.rust-lang.org/en-US/")?;
-
-    Document::from_read(res)?
+
+#[tokio::main]
+
+async fn main() -> Result<(), reqwest::Error>{
+    let res = reqwest::get("https://www.rust-lang.org/en-US/").await?.text().await?;
+
+    Document::from(res.as_str())
         .find(Name("a"))
         .filter_map(|n| n.attr("href"))
         .for_each(|x| println!("{}", x));
 
     Ok(())
 }
+}

```

Comment:
This is causing a failure
error: unexpected closing delimiter: `}`
  --> /tmp/rust-skeptic.v9NBrzp3zOUB/test.rs:20:1
   |
20 | }
   | ^ unexpected closing delimiter
error: aborting due to previous error
test extract_links_sect_extract_all_links_from_a_webpage_html_line_11 ... FAILED